src/oscar/apps/dashboard/catalogue/forms.py

```diff
@@ -220,6 +235,9 @@ def __init__(self, product_class, data=None, parent=None, *args, **kwargs):
             self.instance.product_class = product_class
         self.add_attribute_fields(product_class, self.instance.is_parent)
 
+        if 'slug' in self.fields:
```

Comment:
Is this really something that is part of this PR?

Reply:
Product slug was not previously editable at all, and if we make it editable we have to allow it to be empty (for auto-generation).

Reply:
Yeah, it wasn't there before and there was a need for it.

Reply:
ok